client/my-sites/site-settings/main.jsx

```diff
@@ -104,6 +105,7 @@ export class SiteSettingsComponent extends Component {
 			<Main className="site-settings">
 					<SidebarNavigation />
 					<SiteSettingsNavigation site={ site } section={ section } />
+					<QueryProductsList />
```

Comment:
This one I left here intentionally to load the price before reaching the export page. It should still fire only once when the setting page is loaded, and eventually it should probably be moved to a more global location as more parts of Calypso migrate to reading the products list from the Redux store.